src/components/pacbio/PacbioLibraryCreate.vue

```diff
@@ -125,26 +135,57 @@ export default {
   },
   data() {
     return {
-      library: { tag: { group_id: '' }, sample: {} },
-      tagOptions: [{ value: '', text: 'No tag' }],
+      library: { tag: { id: '' }, sample: {} },
+      selectedTagSetId: '',
+      selectedTagId: '',
     }
   },
   computed: {
-    ...mapGetters('traction', ['tractionTags']),
+    // Map fetched Pacbio tag sets and tags
+    ...mapState('traction/pacbio/poolCreate', {
```

Comment:
I see the ability to reuse VueX methods here and that is something we can look at in the future (implementing something similar for ONT at the moment). But for the pacbio VueX stores I would avoid taking methods/state across modules because that makes a dependency between them. In this instance it may cause side effects to the tag data used in the poolCreate page. I would just copy the vueXmethods from pacbio/poolCreate into pacbio/libraries.

Reply:
I will create a separate vuex module libraryCreate and do them separately.

Reply:
Now LibraryCreate has its own fetch to retrieve tagsets and tags and it writes computed options to its own state. I have replaced mapState with mapGetters as well.